tests/std/tests/VSO_0000000_vector_algorithms/test.cpp

```diff
@@ -271,10 +271,19 @@ auto last_known_good_find_first_of(FwdItH h_first, FwdItH h_last, FwdItN n_first
 
 template <class RanItH, class RanItN>
 auto last_known_good_search(RanItH h_first, RanItH h_last, RanItN n_first, RanItN n_last) {
-    const auto n_len = n_last - n_first;
+    const ptrdiff_t n_len = n_last - n_first;
 
     for (; h_last - h_first >= n_len; ++h_first) {
-        if (equal(h_first, h_first + n_len, n_first, n_last)) {
+        bool is_eq = true;

```

Comment:
This shadows `std::is_eq` from `<compare>`. https://en.cppreference.com/w/cpp/utility/compare/named_comparison_functions

Reply:
I made it `is_eq` in attempt to avoid shadowing, thinking that abbreviation is less likely to shadow.
There's `std::pmr::memory_resource::is_equal`, but probably it does not count, as it is a member.